examples/jsm/controls/OrbitControls.js

```diff
@@ -21,9 +21,11 @@ import {
 const _changeEvent = { type: 'change' };
 const _startEvent = { type: 'start' };
 const _endEvent = { type: 'end' };
-const _ray = new Ray();
-const _plane = new Plane();
-const TILT_LIMIT = Math.cos( 70 * MathUtils.DEG2RAD );
+const _ray = /* @__PURE__ */ new Ray();
+const _plane = /* @__PURE__ */ new Plane();
+
+const { DEG2RAD } = MathUtils;
+const TILT_LIMIT = Math.cos( 70 * DEG2RAD );
```

Comment:
Doesn't this line also need PURE annotation (also, why `MathUtils.DEG2RAD` requires destructuring)?

Reply:
Most built-in JavaScript methods and classes are picked up as pure -- `Object.create` in motion controllers lib was the only exception I found.

I'm not sure why `MathUtils` needed destructuring. I noticed this with https://github.com/mrdoob/three.js/pull/26912#discussion_r1349519605 and previously hardcoded the value until d5ce35ceec8ae5f9fb214f4456350da2e6f55d29. There might have been something similar with reads from `ShaderLib` and `UniformsLib` which I wrapped in IIFEs.

Reply:
Explanation after further investigation in https://github.com/mrdoob/three.js/pull/26912#discussion_r1352129525. `Object.create` is also properly impure (motion controllers lib) since it mutates its arguments. This is consistent with how tools interpret built-ins.